hnn_core/network.py

```diff
@@ -301,33 +303,38 @@ class Spikes(object):
 
     Parameters
     ----------
-    times : list (n_trials,) of list (n_spikes,) of float, shape | None
+    spiketimes : list (n_trials,) of list (n_spikes,) of float, shape | None
```

Comment:
nitpick but `spike_times` is preferred 

if you don't put the underscore it can become confusing where the second word starts. Not a problem here but generally a good practice.